app/assets/javascripts/common/reports.js

```diff
@@ -645,69 +582,32 @@ DashboardReports = () => {
 
       return config;
     },
-    lostToFollowUpTrend: function(data) {
-      const config = createBaseGraphConfig();
-      config.data = {
+    lostToFollowUpTrend: function (data) {
+      const additionalLostToFollowUpTrendConfig = {
+        data: {
           labels: Object.keys(data.ltfuPatientsRate),
           datasets: [
-              {
-                  label: "Lost to follow-up",
-            backgroundColor: colors.lightBlue,
-            borderColor: colors.darkBlue,
-                  borderWidth: 2,
-            pointBackgroundColor: colors.white,
-            hoverBackgroundColor: colors.white,
-                  hoverBorderWidth: 2,
-                  data: Object.values(data.ltfuPatientsRate),
-                  type: "line",
-              },
-          ],
-      };
-      config.options.scales = {
-          xAxes: [
-              {
-                  stacked: false,
-                  display: true,
-                  gridLines: {
-                      display: false,
-                      drawBorder: true,
-                  },
-                  ticks: {
-                      autoSkip: false,
-              fontColor: colors.darkGrey,
-                      fontSize: 12,
-                      fontFamily: "Roboto",
-                      padding: 8,
-                      min: 0,
-                      beginAtZero: true,
-                  },
-              },
-          ],
-          yAxes: [
-              {
-                  stacked: false,
-                  display: true,
-                  gridLines: {
-                      display: true,
-                      drawBorder: false,
-                  },
-                  ticks: {
-                      autoSkip: false,
-              fontColor: colors.darkGrey,
-                      fontSize: 10,
-                      fontFamily: "Roboto",
-                      padding: 8,
-                      min: 0,
-                      beginAtZero: true,
-                      stepSize: 25,
-                      max: 100,
-                  },
-              },
+            {
+              label: "Lost to follow-up",
+              backgroundColor: colors.lightBlue,
+              borderColor: colors.darkBlue,
+              borderWidth: 2,
+              pointBackgroundColor: colors.white,
+              hoverBackgroundColor: colors.white,
+              hoverBorderWidth: 2,
+              data: Object.values(data.ltfuPatientsRate),
+              type: "line",
+            },
           ],
+        },
       };
+      const config = combineConfigWithBaseConfig(
```

Comment:
The juggling could be simplified a bit. I'd say you can call the custom config in each of these methods simply `config` (it should be evident that it's the config related to the method in question).

Then, you can simply do a `return combineConfigWithBaseConfig(config)`, instead of calling the method here and doing the return below.

Reply:
Great suggestion, I will do this with the configs

Note: there will be 2 graphs where this is currently not possible until after the upgrade due to how the 'scales' object is handled in this version - I will include these in a tidy up for a future PR